### PR TITLE
fix: use the correct maxWidth for OTPInput

### PR DESCRIPTION
**Pull request by chaitanyadeorukhkar**

### CODE REVIEW

File: packages/blade/src/components/Input/OTPInput/OTPInput.tsx [hunk 1]
```diff
@@ -210,9 +212,9 @@ const OTPInput = ({
       inputs.push(
         <Box
           flex={1}
-          paddingLeft={index == 0 ? 'spacing.0' : 'spacing.3'}
+          marginLeft={index == 0 ? 'spacing.0' : 'spacing.3'}
           key={`${inputId}-${index}`}
-          maxWidth={100} // TODO: Get the correct value for this from design
+          maxWidth={platform === 'onDesktop' ? 36 : 40} // TODO: use size tokens
```
Comment (saurabhdaware) [inline packages/blade/src/components/Input/OTPInput/OTPInput.tsx:217]: are we doing this TODO in followup PR?

Comment (chaitanyadeorukhkar) [reply packages/blade/src/components/Input/OTPInput/OTPInput.tsx:217]: No, keeping it there cuz we need to add size tokens to Blade which we don't have right now. So across our components we've hardcoded sizes for now😢

Comment (divyanshu013) [reply packages/blade/src/components/Input/OTPInput/OTPInput.tsx:217]: Can create an issue for tracking this.

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

**Review by saurabhdaware — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**